src/api/rootResolver.js

```diff
@@ -1,11 +1,43 @@
 import merge from 'lodash/merge'
-import eventsList from '../fixtures/events.json'
+import PublicDeployer from '@noblocknoparty/contracts/build/contracts/Deployer.json'
+import PrivateDeployer from '../build/contracts/Deployer.json'
+import eventsList from '../fixtures/events.json'  
+import {toHex, toWei} from 'web3-utils'
 import getEthers, { signer } from './ethers'
 import singleEventResolvers, {
   defaults as singleEventDefaults
 } from './resolvers/singleEventResolvers'
 import ensResolvers, { defaults as ensDefaults } from './resolvers/ensResolvers'
 
+const deployerAbi = PublicDeployer.abi
+const deployerContractAddresses = Object.assign(
+  {},
+  PrivateDeployer.networks,
+  PublicDeployer.networks
+)
+
+function getNetwork(){
+  return new Promise(function(resolve,reject){
+    window.web3.version.getNetwork(function(err, result){
+      resolve(result);
+    });
+  });
+}
+
+function getEvents(){
```

Comment:
Can we abstract this to a separate file in something like `/src/api/getEvents` and then we can get rid of all the extraneous imports.

We should also probably create an 'eventsResolver' to keep the root resolver cleaner. See how I'm doing it for the singleEventsResolver